engine/tutorials/networkingcontainers.md

```diff
@@ -33,6 +33,8 @@ The network named `bridge` is a special network. Unless you tell it otherwise, D
 
     74695c9cea6d9810718fddadc01a727a5dd3ce6a69d09752239736c030599741
 
+<center>![](bridge1.png)</center>
```

Comment:
Also, please add alt text (between the square brackets) whenever you add an image. For example:

```markdown
![my pretty image](my_pretty_image.png)
```